creusot/tests/should_succeed/hillel.mlcfg

```diff
@@ -42,7 +42,7 @@ module Core_Num_Impl11_Max
   use prelude.Int
   use prelude.UIntSize
   let constant mAX'  : usize = [@vc:do_not_keep_trace] [@vc:sp]
-    (18446744073709551615 : usize)
+    [#"/rustc/1716932743a7b3705cbf0c34db0c4e070ed1930d/library/core/src/num/uint_macros.rs" 49 8 49 27] (18446744073709551615 : usize)
```

Comment:
We might as well not include these paths as they are unusable in why3

Reply:
As you wish. I thought this would be better than nothing.

Reply:
I could make these paths relative to the homedir. But do you think that it would stable enough? I.e., is that path of the Rust stdlib sources always the same in the homedir?

Reply:
The path of rust sources is stable to the sysroot. In practice this is the same as being stable to the home directory

Reply:
Alright... since Why3 does not support the ~ syntax for paths, I chose to simply discard those spans...